modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -152,10 +151,10 @@ public void recordingTestThatShouldHaveCorrectDuration() throws IOException, Int
                     .start();
                 String ffmpegOutput = container.getLogs();
 
-                assertTrue(
-                    "Duration is incorrect in:\n " + ffmpegOutput,
-                    ffmpegOutput.contains("Duration: 00:") && !(ffmpegOutput.contains("Duration: 00:00:00.00"))
-                );
+                assertThat(ffmpegOutput)
+                    .as("Duration is incorrect in:\n " + ffmpegOutput)
```

Comment:
```suggestion
                    .as("Duration starts with 00:")
```

I think that's the idea of the test 😅